layers/+readers/pdf/README.org

```diff
@@ -101,43 +101,45 @@ If you use Emacs editing style, check the key bindings at the [[https://github.c
 
 ** PDF View
 
-| *Key binding*        | *Description*                             |
-|----------------------+-------------------------------------------|
-| *Navigation*         |                                           |
-|----------------------+-------------------------------------------|
+| *Key binding*      | *Description*                               |
```

Comment:
The headlines are not longer correctly aligned. Please align the headlines like below:
```
|----------------------+-------------------------------------------|
| *Navigation*         |                                           |
|----------------------+-------------------------------------------|
```

Reply:
Hi, of course I would like to align the table correctly but this is how org-mode aligns them. Of course I just used TAB to let org-mode align the table automatically, so on my system this file shows like shown in the image below
![image](https://user-images.githubusercontent.com/18429791/79700761-663af080-8298-11ea-8933-e59b0287d29e.png)
Indeed when I switch to text mode it looks like the table in the text file.
When I align the table in the text file and switch to org-mode then the table looks like in the following image
![image](https://user-images.githubusercontent.com/18429791/79701088-879cdc00-829a-11ea-9ba1-5336b685c13c.png)
But I guess the alignment is not very important as the table looks fine here on github anyway.
If my org-mode does the alignment wrong then let me know (and also let me know if you know a solution). Also, I am fine to push the manually text-mode aligned table if prefered